src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,28 @@
-export const TodoInfo = () => {};
+import { FC } from 'react';
+import cn from 'classnames';
+
+import { TodoWithUser } from '../../types';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: TodoWithUser;
+};
+
+export const TodoInfo: FC<Props> = ({ todo }) => {
+  const { id, title, user } = todo;
+
+  return (
+    <article
+      data-id={id}
+      className={cn({
+        TodoInfo: true,
+        'TodoInfo--completed': todo.completed,
+      })}
```

Comment:
```suggestion
      className={cn(
        'TodoInfo',
        {
          'TodoInfo--completed': completed,
        },
      )}
```
Static classes can be passes as strings